tests/dist/python/create_part_graph.py

```diff
@@ -0,0 +1,18 @@
+import dgl
```

Comment:
why do we need an explicit script instead of a utility function for graph partition?